src/frontends/pytorch/src/transforms/aten_index_replacer.cpp

```diff
@@ -223,8 +219,9 @@ AtenIndexToSelect::AtenIndexToSelect() {
             }
             auto final_shape = std::make_shared<v0::Concat>(concat_dims, 0);
             gather = std::make_shared<v1::Reshape>(gather, final_shape, false);
-            copy_runtime_info({index_op, input_node, indicies}, gather);
+            copy_runtime_info({index_op, indicies}, gather);
             replace_node(index_op, gather);
+            gather->set_friendly_name(index_op->get_friendly_name());
```

Comment:
Could you inherit `aten::index` name for other cases too? This really helps to debug problems